src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Import/ImportResourceParser.cs

```diff
@@ -67,7 +72,7 @@ private static void CheckConditionalReferenceInResource(Resource resource)
 
                 if (reference.Reference.Contains('?', StringComparison.Ordinal))
                 {
-                    throw new NotSupportedException("Conditional reference is not supported for $import.");
+                    throw new NotSupportedException("Conditional reference is not supported for initial $import.");
```

Comment:
My only suggestion would be to use the enum name vs "initial" in this string. If more import modes are added later, this won't have to be updated.

Reply:
Fixed.